ui-ngx/src/app/modules/home/components/widget/lib/analogue-gauge.models.ts

```diff
@@ -864,6 +864,7 @@ export abstract class TbBaseGauge<S, O extends GenericOptions> {
         }
       }
     }
+    this.gauge.update({} as GenericOptions);
```

Comment:
This code updates the status of the library state, but for better performance widgets, you need to understand why the rendering is not updated when changing the value.